ui/lib/apps/TopSQL/pages/List/List.tsx

```diff
@@ -282,34 +225,47 @@ export function TopSQLList() {
   )
 }
 
-const useTopSQLData = () => {
+const computeTimeWindowSize = (
+  screenWidth: number,
+  [min, max]: TimeRangeValue
+) => {
+  const barWidth = 8
+  const windowSize = Math.ceil((barWidth * (max - min)) / screenWidth)
+  return windowSize
+}
+
+const useTopSQLData = (
+  containerRef: React.RefObject<HTMLDivElement>,
+  instance,
+  timeRange
+) => {
   const [topSQLData, setTopSQLData] = useState<TopsqlSummaryItem[]>([])
+  const [timeWindowSize, setTimeWindowSize] = useState(0)
   const [isLoading, setIsLoading] = useState(false)
   const updateTopSQLData = useCallback(
-    async ({
-      instance,
-      timestamps,
-      timeWindowSize,
-      topN,
-    }: {
-      instance: TopsqlInstanceItem
-      timestamps: [number, number]
-      timeWindowSize: number
-      topN: number
-    }) => {
+    async (_instance: TopsqlInstanceItem | null, _timeRange: TimeRange) => {
```

Comment:
just curious, in what case will you add a `_` prefix? It is usually used to indicate the parameter is not used I think.

Reply:
emm.. I use it to implicit an internal variable if there is the same variable outside. So that the variables won't be missed when refactor.